apps/files/src/components/AcceptInvite.vue

```diff
@@ -0,0 +1,79 @@
+  <template>
+    <div id="files" class="uk-flex uk-flex-column">
+     <accept-invite-bar/>
+  <div name="accept-invite-dialog">
+      <oc-text-input
+        :disabled="$_ocLoading"
+        :placeholder="$_acceptInviteDialogPlaceholder"
+        :label="$_acceptInviteDialogLabel"
+        autofocus
+        id="accept-invite-input"
+        v-model="inputValue"
+        ref="input"
+        @keydown.enter.native="onConfirm"
+      ></oc-text-input>
+        <oc-button id="accept-invite-cancel" @click.stop="onCancel">{{ _ocCancelText }}</oc-button>
+        <oc-button :disabled="$_ocLoading || inputValue === '' || clicked"
+               id="accept-invite-ok"
+               ref="confirmButton"
+               @click.stop="onConfirm">{{ _ocConfirmText }}</oc-button>
+  </div>
+  </div>
+</template>
+<script>
+import AcceptInviteBar from './AcceptInviteBar.vue'
+import { mapActions } from 'vuex'
+
+export default {
+  components: {
+    AcceptInviteBar
+  },
+  data: () => ({
+    acceptInviteName: '',
+    inputValue: null,
+    clicked: false,
+    fileFolderCreationLoading: false
+  }),
+  computed: {
+    $_ocLoading () {
+      return this.fileFolderCreationLoading
+    },
+    $_acceptInviteDialogLabel () {
+      return this.$gettext('Token')
+    },
+    $_acceptInviteDialogPlaceholder () {
+      return this.$gettext('Enter your token…')
+    },
+    _ocConfirmText () {
```

Comment:
When having private property, pls use `$_` prefix instead of only `_`

https://vuejs.org/v2/style-guide/#Private-property-names-essential